windows/vhba_hot_add_remove/enable_vm_nvme_spec13.yml

```diff
@@ -13,7 +13,23 @@
     set_win_power_state: "shutdown"
 
 # Add extra config for VM
-- include_tasks: ../../common/vm_set_extra_config.yml
+- name: "Set 'nvme.specVersion' with '103' in VM's extra config"
+  include_tasks: ../../common/vm_set_extra_config.yml
+
+- name: "Power on VM"
+  include_tasks: ../../common/vm_set_power_state.yml
+  vars:
+    vm_power_state_set: 'powered-on'
+
+- name: "Get VM's extra config"
+  include_tasks: ../../common/vm_get_extra_config.yml
+
+- name: "Check 'nvme.specVersion' is '103' in VM's extra config after VM power-on"
+  ansible.builtin.assert:
+    that:
+      - vm_extra_config['nvme.specVersion'] is defined
+      - vm_extra_config['nvme.specVersion'] == '103'
+    fail_msg: "The 'nvme.specVersion' is not correct in VM's extra config after VM power-on"
```

Comment:
The fail msg can be more specific that checked extra config value is not 103.

Reply:
Updated msg